test/NuGet.Core.Tests/NuGet.Common.Test/DateTimeUtilityTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    public class DateTimeUtilityTests
```

Comment:
This class looks unrelated the issue.

Reply:
I have no idea what's going on. None of these are files I changed....

> git diff dev --name-only
> test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs
> test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

Reply:
There we go, must have been a bad rebase somehow.